statistics/update_test.go

```diff
@@ -456,53 +456,45 @@ func (s *testStatsUpdateSuite) TestQueryFeedback(c *C) {
 			// test primary key feedback
 			sql: "select * from t where t.a <= 5",
 			hist: "column:1 ndv:3 totColSize:0\n" +
-				"num: 1\tlower_bound: 1\tupper_bound: 1\trepeats: 1\n" +
+				"num: 1\tlower_bound: -9223372036854775808\tupper_bound: 1\trepeats: 0\n" +
 				"num: 2\tlower_bound: 2\tupper_bound: 2\trepeats: 1\n" +
-				"num: 4\tlower_bound: 3\tupper_bound: 6\trepeats: 0",
+				"num: 4\tlower_bound: 3\tupper_bound: 5\trepeats: 0",
 			idxCols: 0,
 		},
 		{
 			// test index feedback by double read
 			sql: "select * from t use index(idx) where t.b <= 5",
 			hist: "index:1 ndv:2\n" +
-				"num: 2\tlower_bound: 2\tupper_bound: 2\trepeats: 2\n" +
+				"num: 2\tlower_bound: \tupper_bound: 2\trepeats: 0\n" +
 				"num: 4\tlower_bound: 3\tupper_bound: 6\trepeats: 0",
 			idxCols: 1,
 		},
 		{
 			// test index feedback by single read
 			sql: "select b from t use index(idx) where t.b <= 5",
 			hist: "index:1 ndv:2\n" +
-				"num: 2\tlower_bound: 2\tupper_bound: 2\trepeats: 2\n" +
+				"num: 2\tlower_bound: \tupper_bound: 2\trepeats: 0\n" +
```

Comment:
Why the lower bound is empty?

Reply:
Because for `t.b <= 5` using index scan, the low value of the scan range is `null`.